jaxlib/pytree.cc

```diff
@@ -158,6 +158,9 @@ class PyTreeDef {
   bool operator!=(const PyTreeDef& other) const { return !(*this == other); }
 
   std::string ToString() const;
+  
+  py::tuple GetTraversal() {return py::make_tuple(traversal_); }
+  void SetTraversal(py::tuple t) { traversal_ = t.cast<std::vector<Node>>(); }
```

Comment:
yes i like that idea, how would you add that... just unflatten / flatten to pickle / unpickle? 

one issue is, the pytreedef would need data in order to be pickled. potentially it would be possible to just put shapedarrays or placeholders there?